### PR TITLE
Fix variable name to follow proper convention

### LINKED ISSUE #60: Convert all field names and variable names to use CamelCase instead of _

### PR DESCRIPTION
**Pull request by X3Technology**
This PR refactors all of the table names, column names, variable name to follow the proper naming conventions.  Almost every file is touched in some way....

resolves #60

### CODE REVIEW

**Comment by WillStrohl** [comment]
I haven't had as much experience with this code as some of you, so I'm wondering something that most of you may already know the answer to...  

Why do we have two different installation scripts?  

One is the `DAL\Install.SqlDataProvider` and is lacking all of the updates in the `DBScripts\00.01.00.SqlDataProvider` file that we've been working on.  

[EDIT]
FYI - The manifest is only looking at the `DBScripts` version.  
[/EDIT]

**Comment by X3Technology** [comment]
Hold off on that.  The taes have changed significantly...

Sent via the Samsung Galaxy S20 5G, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by david-poindexter** [comment]
This PR is going to need to be rebased (due to changes to `DBScripts/00.01.00.SqlDataProvider`) before it can be merged.

File: Dnn.StructuredContent.csproj
Comment (valadas) [inline Dnn.StructuredContent.csproj]: This is needed by stylecop

**Review by valadas — APPROVED** [review]
Looks good to me

**Pull request merged**
